NineChronicles/2023-10/PlanetSpec.schema.json

```
@@ -57,6 +57,27 @@
                         "format": "uri"
                     }
                 },
+                "world-boss.gql": {
+                    "type": "array",
+                    "items": {
+                        "type": "string",
+                        "format": "uri"
+                    }
+                },
+                "patrol-reward.gql": {
+                    "type": "array",
+                    "items": {
+                        "type": "string",
+                        "format": "uri"
+                    }
+                },
+                "market.gql": {
```

Maybe the *market* means `planetarium/market-service` repository. I remember it provides REST API instead of GraphQL. (https://github.com/planetarium/market-service/blob/main/MarketService/Controllers/MarketController.cs) cc @ipdae 

```suggestion
                "market.rest": {
```